cirq/experiments/qubit_characterizations.py

```diff
@@ -0,0 +1,567 @@
+import itertools
+from typing import Sequence, Dict
+from cirq import circuits, devices, ops, protocols, sim, study, value
+import numpy
+from matplotlib import pyplot
+from mpl_toolkits.mplot3d import Axes3D
+
+
+class RabiExperimentResult(object):
+    """Class for storing and plotting data from a Rabi oscillation
+    experiment.
+    """
+
+    def __init__(self, num_shots: int, rabi_angles: Sequence[float],
+                 excited_state_pops: Sequence[float]):
+        """
+        Args:
+            num_shots: The number of repetitions of the circuit for each Rabi
+              angle.
```

Comment:
Nit: indent four spaces for line continuation of arg docs.

Reply:
Done